humble/distribution.yaml

```diff
@@ -5454,6 +5454,24 @@ repositories:
       url: https://github.com/ros-visualization/python_qt_binding.git
       version: humble
     status: maintained
+  qbshin-ros:
+    release:
+      packages:
+      - qb_softhand_industry
+      - qb_softhand_industry_description
+      - qb_softhand_industry_driver
+      - qb_softhand_industry_msgs
+      - qb_softhand_industry_ros2_control
+      - qb_softhand_industry_srvs
+      tags:
+        release: release/humble/{package}/{version}
+      url: https://bitbucket.org/qbrobotics/qbshin-ros2-release.git
```

Comment:
This URL is allowed for doing releases, but note that if you ever want to release into Rolling this will have to move into the https://github.com/ros2-gbp organization.

Reply:
Got it! However we are not interested in release our packages on Rolling